### PR TITLE
Add more definition to the spec sheet

### PR DESCRIPTION
**Pull request by owalk**

### CODE REVIEW

File: docs/specSheet.md [hunk 1]
```diff
@@ -35,6 +37,12 @@ Lunk always goes forward, but he can choose to turn to the left of the right.
 
 If Lunk falls off the map, he jumps back up so high that the landing spot is a random area within a 10 by 10 area near the fall point
 
+### Wall Collisions
+If Lunk initiates a wall collision the following will take effect:
+* Lunk pauses movement for 1 second while slowly turning around
+* Lunk chooses a new direction to run from a random direction not facing the wall
+* Lunk resumes running
+
```
Comment (Sean-Nishi) [inline docs/specSheet.md:45]: In the Lunk videos when he "interacts" with a wall he destroys it by walking through it. We could implement that but give a huge penalty to the player.

**Review by Sean-Nishi — COMMENTED** [review]
See Wall Collisions comment..

File: docs/specSheet.md (hunk 1 above)
Comment (owalk) [reply docs/specSheet.md:45]: There is a roadblock to getting the art assets for that implementation. 

The pause-and-turn approach adds more of a character persona to the game. Maybe we can run through the walls sometimes but have this as an additional feature added after the roadblock is resolved.

Comment (owalk) [reply docs/specSheet.md:45]: @Sean-Nishi Can you summarize the effect on the gameplay of your preferred approach and compare it to my suggested approach?

Example: If strictly breaking walls is in place the map will change as lunk moves. This can be used to create interesting situations. What would one of those situations be and how would it be implemented?

Comment (owalk) [reply docs/specSheet.md:45]: notes from call:

Delete the wall for now, no animation.
Remove 25% (some large value, extreme detriment) of total hp/boredom meter
Make it 50/50 percent chance of what the action is, randomly destroy wall or turn around

**Comment by owalk** [comment]
change bored bar to boredom meter in new commit

File: docs/specSheet.md [hunk 2]
```diff
@@ -83,4 +91,9 @@ Lunk gains points for doing things he likes to do.
 Each favorable action adds to the total high score for an amount specific to each action
 
 ### Bored Bar
```
Comment (Sean-Nishi) [inline docs/specSheet.md:93]: Boredom Meter

**Review by Sean-Nishi — APPROVED** [review]
Looks good!
